composer.json

```diff
@@ -72,6 +72,14 @@
             ],
             "locations": {
                 "web-root": "web/"
+            },
+            "file-mapping": {
+            "[web-root]/sites/development.services.yml": {
+                "mode": "replace",
+                "path": "web/core/assets/scaffold/files/development.services.yml",
+                "overwrite": false
+            },
+            "allowed-packages": ["ramsalt/drupal-scaffold"]
```

Comment:
I might have messed it up, but this line is now already part of the composer.json

Reply:
Removed the duplicate